cli/tauri.js/src/template.ts

```diff
@@ -25,20 +34,33 @@ const injectConfFile = (injectPath, { force, logging }) => {
   }
 }
 
-const injectTemplate = (injectPath, { force, logging, tauriPath }) => {
+const injectTemplate = (
+  injectPath: string,
+  {
+    force,
+    logging,
+    tauriPath
+  }: {
```

Comment:
Not a must fix, more my opinion, but I think it would be good to decide on the Tauri style in general when it comes to using inline type definitions for function parameters.  Although allowed, I generally avoid doing so as I find it makes the code less readable/maintainable, especially to the TS novice.  I generally recommend defining an explicit interface for the type so in this case `InjectTemplateParams` or something like it.  It also helps to avoid possible duplicate type definitions as it is easier to reuse if need in the future, vs just defining the same type again.

Reply:
Done! Thanks for the suggestion! 437ce947f165a0db38b6383c9d25afc0d26cc274